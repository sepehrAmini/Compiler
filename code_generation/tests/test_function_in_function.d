void calc2(){
    Print("im in CALC2 function");
}

void calc(){
    calc2();
}

int main(){
    calc();
}