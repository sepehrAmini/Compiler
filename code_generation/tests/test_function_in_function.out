im in CALC2 function